Neos.Cache/Classes/Backend/MultiBackend.php

```diff
@@ -83,40 +74,34 @@ protected function prepareBackends(): void
     }
 
     /**
-     * @param string $backendClassName
-     * @param array $backendOptions
-     * @return BackendInterface
-     * @throws \Throwable
+     * @throws Throwable
      */
     protected function buildSubBackend(string $backendClassName, array $backendOptions): ?BackendInterface
     {
-        $backend = null;
         try {
             $backend = $this->instantiateBackend($backendClassName, $backendOptions, $this->environmentConfiguration);
             $backend->setCache($this->cache);
-        } catch (\Throwable $t) {
-            $this->handleError($t);
+        } catch (Throwable $throwable) {
+            $this->logger && $this->throwableStorage && $this->logger->error(sprintf('Failed creating sub backend %s for %s: %s', $backendClassName, get_class($this), $this->throwableStorage->logThrowable($throwable)), LogEnvironment::fromMethodName(__METHOD__));
```

Comment:
Or, since this is used in many places, maybe extract it to a private method!?

Reply:
I used `$this->logger && $this->logger->info()` all the time in my code, I tend to like it.

In this case, I added `&& $this->throwableStorage` only because Psalm complained. Which is only theoretical, because there can't be a logger if there's no Throwable Storage as per the constructor code.

But yeah, I'll put it into a method.

Reply:
just `$this->logger?->info()` with php 8 ^^

Reply:
But thinking about it … if we put that into a private method, how would that look like? If that method logs, we loose the context information (LogEnvironment). And it's not always "error", it's also "warn". And sometimes it's `sprintf()`ed.

It's really just a Psalm thing.

Reply:
I'll try that.